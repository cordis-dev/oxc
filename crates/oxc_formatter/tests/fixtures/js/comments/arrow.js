() =>
  // comment
  [];

() =>
  // comment
  ({});

() /* comment1 */ => /* comment2 */ { };

() /**/ => //
  () /**/ => /**/
    () /**/ => /**/ {
      //
    }

call(
  () /* comment1 */ => /* comment2 */ { }
);

call(
  () /**/ => //
    () /**/ => /**/
      () /**/ => /**/ {
        //
      }
);
