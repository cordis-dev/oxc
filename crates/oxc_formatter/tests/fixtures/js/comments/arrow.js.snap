---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
() =>
  // comment
  [];

() =>
  // comment
  ({})

==================== Output ====================
() =>
  // comment
  [];

() =>
  // comment
  ({});

===================== End =====================
